src/index.ts

```diff
@@ -463,7 +470,7 @@ const main = async (baseInfo: LSPluginBaseInfo) => {
       title: 'Enter the template to use for new highlights',
       description:
         'Enter the template to use for new highlights. Required variables are: {{{text}}}, {{{highlightUrl}}}. Optional variables are {{{dateHighlighted}}}',
-      default: `> {{{text}}} [⤴️]({{{highlightUrl}}})`,
+      default: `> {{{text}}} [⤴️]({{{highlightUrl}}}) {{#labels}}#[[{{{name}}}]]{{/labels}}`,
```

Comment:
I guess we could add a space in between the labels otherwise it could be a little squeezy if there are many

Reply:
<img width="690" alt="Screenshot 2022-12-30 at 4 10 58 PM" src="https://user-images.githubusercontent.com/12963965/210048841-7e719b80-29d8-4852-8782-2a2ae7f0cf2c.png">

Reply:
Good call on the space, my theme kind of had some padding so I didn't notice.